README.md

```diff
@@ -1,14 +1,87 @@
-# Sinco
```

Comment:
would need to revert this back to the original unfortunately, as all drashland projects should have a standardised README

Reply:
Guessing this was re-added in a merge conflict actually